bin/docker-entrypoint

```diff
@@ -4,9 +4,10 @@ set -e
 worker() {
   WORKERS_COUNT=${WORKERS_COUNT:-2}
   QUEUES=${QUEUES:-queries,scheduled_queries,celery,schemas}
+  MAX_MEMORY=$(($(/usr/bin/awk '/MemTotal/ {print $2}' /proc/meminfo)/4))
```

Comment:
I'm not sure this is a good default to have. 

Maybe just add the option to pass additional options to `celery worker` command? Then you can pass this one or whatever else you might need.

Reply:
Sure.